client/components/AssignDoi/assignDoiPreview.scss

```diff
@@ -0,0 +1,44 @@
+.assign-doi-preview-component {
+	margin: 12px 0;
+	padding: 12px;
+	border: 1px dashed #ccc;
+	border-radius: 3px;
+	font-size: 0.9em;
+
+	dl {
+		display: inline-grid;
+		grid-template-columns: auto auto;
+		margin: 0;
+
+		&:not(:last-child) {
+			margin: 0 0 18px 0;
+		}
+
+		> dt {
```

Comment:
What do you think about a `text-align: right` on the `dt` elements? Gives the whole thing a bit more visual structure IMO.

Reply:
I like it!